### ISSUE TITLE
Failed to run with node 16

### ISSUE THREAD

**Issue by KnisterPeter** [nolanlawson/marky#26]
When running with node 16, we'll get this error:

```
TypeError: perf.getEntriesByName is not a function
    at Object.exports.stop (/app/node_modules/marky/lib/marky.cjs.js:55:24)
    at Function.timeEnd (/app/node_modules/lighthouse-logger/index.js:128:11)
    at Function.requireGatherers (/app/node_modules/lighthouse/lighthouse-core/config/config.js:626:9)
    at new Config (/app/node_modules/lighthouse/lighthouse-core/config/config.js:258:27)
    at generateConfig (/app/node_modules/lighthouse/lighthouse-core/index.js:65:10)
    at lighthouse (/app/node_modules/lighthouse/lighthouse-core/index.js:43:18)
    at runLighthouse (/app/node_modules/lighthouse/lighthouse-cli/run.js:226:32)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
```

Are you open for a PR?

**Comment by brendankenny** [comment]
I believe this is the same as #18 and was fixed in #19

**Comment by KnisterPeter** [comment]
Oh thanks. Search for that but didn't found it. I'll close this one and sorry for the extra work.
